src/test/java/com/emc/mongoose/storage/driver/pravega/integration/CommonTest.java

```diff
@@ -126,7 +125,7 @@ public CommonTest ()
 
 		private CommonTest (final Config config)
 			throws OmgShootMyFootException {
-			super(
+			pravegaStorageDriver = new PravegaStorageDriver(
```

Comment:
looks to be not needed

Reply:
It's a stub that should test our driver in the future, if I understand correctly. So we should have an instance of driver. So we call its constructor. Can you please expand on what's wrong here?

Reply:
yep. then it's fine